template/python-3/Dockerfile

```
@@ -0,0 +1,43 @@
+FROM --platform=${TARGETPLATFORM:-linux/amd64} ghcr.io/polyxia-org/alpha:main as get-alpha
```

```suggestion
FROM --platform=${TARGETPLATFORM:-linux/amd64} ghcr.io/polyxia-org/alpha:v0.2.1 as get-alpha
```
Should use static version

---

fixed